frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -172,28 +171,26 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     type,
   });
 
```

Comment:
imo it still has a value to have vmName inside the DV name. You can easily see which PVCs belong to which VM this way.

Although, it will stop making sense, when you have collisions as it will show the old disk which this VM is currently not using.

thoughts on what is better?

Reply:
I definitely find it convenient to be able to identify the PVCs using the VM name. I don't know how often the collisions case will occur, but I feel like the convenience outweighs the corner case.